picoquic/sender.c

```diff
@@ -853,6 +853,11 @@ void picoquic_cnx_set_next_wake_time(picoquic_cnx_t* cnx, uint64_t current_time)
                 }
             }
         }
+
+        /* Consider keep alive */
+        if (cnx->keep_alive_interval) {
+            next_time = next_time < cnx->keep_alive_interval ? next_time : cnx->keep_alive_interval;
+        }
```

Comment:
Should this be a direct comparison to the interval, or to "current_time + cnx->keep_alive_interval" ?